Builds failing due to linuxtools.docker.feature

Time to update the target platform definition again?

```
14199 [INFO] Fetching content.jar from http://download.eclipse.org/linuxtools/update-docker/ (58.42kB)
14201 [INFO] Fetching content.jar from http://download.eclipse.org/linuxtools/update-docker/ (58.42kB)
14362 [ERROR] Failed to resolve target definition /home/travis/build/GoogleCloudPlatform/google-cloud-eclipse/eclipse/photon/gcp-eclipse-photon.target: Could not find "org.eclipse.linuxtools.docker.feature.feature.group/4.0.0.201806122135" in the repositories of the current location -> [Help 1]
```

--- a/eclipse/eclipse-2018-09/gcp-eclipse-2018-09.target
+++ b/eclipse/eclipse-2018-09/gcp-eclipse-2018-09.target
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/mbarbero/fr.obeo.releng.targetplatform -->
-<target name="GCP for Eclipse 2018-09" sequenceNumber="1536244966">
+<target name="GCP for Eclipse 2018-09" sequenceNumber="1537534757">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
       <unit id="org.eclipse.sdk.feature.group" version="4.9.0.v20180830-1030"/>
@@ -31,7 +31,7 @@
       <repository location="http://download.eclipse.org/webtools/downloads/drops/R3.11.0/S-3.11.0.M2-20180803074505/repository"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.linuxtools.docker.feature.feature.group" version="4.0.0.201806122135"/>
+      <unit id="org.eclipse.linuxtools.docker.feature.feature.group" version="4.0.1.201809121533"/>
       <repository location="http://download.eclipse.org/linuxtools/update-docker"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">

--- a/eclipse/oxygen/gcp-eclipse-oxygen.target
+++ b/eclipse/oxygen/gcp-eclipse-oxygen.target
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/mbarbero/fr.obeo.releng.targetplatform -->
-<target name="GCP for Eclipse Oxygen" sequenceNumber="1536244907">
+<target name="GCP for Eclipse Oxygen" sequenceNumber="1537534561">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
       <unit id="org.eclipse.sdk.feature.group" version="4.7.3.v20180330-0919"/>
@@ -31,7 +31,7 @@
       <repository location="http://download.eclipse.org/webtools/downloads/drops/R3.9.3/R-3.9.3-20180302093744/repository"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.linuxtools.docker.feature.feature.group" version="4.0.0.201806122135"/>
+      <unit id="org.eclipse.linuxtools.docker.feature.feature.group" version="4.0.1.201809121533"/>
       <repository location="http://download.eclipse.org/linuxtools/update-docker"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">

--- a/eclipse/photon/gcp-eclipse-photon.target
+++ b/eclipse/photon/gcp-eclipse-photon.target
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/mbarbero/fr.obeo.releng.targetplatform -->
-<target name="GCP for Eclipse Photon" sequenceNumber="1536244897">
+<target name="GCP for Eclipse Photon" sequenceNumber="1537534540">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
       <unit id="org.eclipse.sdk.feature.group" version="4.8.0.v20180611-0826"/>
@@ -31,7 +31,7 @@
       <repository location="http://download.eclipse.org/webtools/downloads/drops/R3.10.0/R-3.10.0-20180611164516/repository"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.linuxtools.docker.feature.feature.group" version="4.0.0.201806122135"/>
+      <unit id="org.eclipse.linuxtools.docker.feature.feature.group" version="4.0.1.201809121533"/>
       <repository location="http://download.eclipse.org/linuxtools/update-docker"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">

--- a/plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/FlexMavenPackagedProjectStagingDelegateTest.java
+++ b/plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/FlexMavenPackagedProjectStagingDelegateTest.java
@@ -33,12 +33,15 @@ import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.NullProgressMonitor;
 import org.eclipse.swtbot.swt.finder.junit.SWTBotJunit4ClassRunner;
 import org.junit.Assume;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 @RunWith(SWTBotJunit4ClassRunner.class)
 public class FlexMavenPackagedProjectStagingDelegateTest {
 
+  // flaking on Photon: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/3256
+  @Ignore
   @Test
   public void testStage_springBoot() throws IOException, CoreException {
     Assume.assumeTrue("Only for JavaSE-8", JavaRuntimeUtils.hasJavaSE8());

--- a/plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportMavenAppEngineStandardProjectTest.java
+++ b/plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportMavenAppEngineStandardProjectTest.java
@@ -37,6 +37,7 @@ import org.eclipse.wst.common.project.facet.core.IFacetedProject;
 import org.eclipse.wst.common.project.facet.core.IProjectFacetVersion;
 import org.eclipse.wst.common.project.facet.core.ProjectFacetsManager;
 import org.junit.Assume;
+import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
@@ -53,6 +54,8 @@ public class ImportMavenAppEngineStandardProjectTest extends BaseProjectTest {
   @Rule
   public TemporaryFolder tempFolder = new TemporaryFolder();
 
+  // flaking on Photon: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/3256
+  @Ignore
   @Test
   public void runImport() throws IOException, CoreException {
     Assume.assumeTrue("No JavaSE 8 JRE found", JavaRuntimeUtils.hasJavaSE8());

--- a/pom.xml
+++ b/pom.xml
@@ -597,7 +597,6 @@
               </execution>
             </executions>
             <configuration>
-              <verbose>true</verbose>
               <source>8</source>
               <target>8</target>
               <compilerArgs>

--- a/pom.xml
+++ b/pom.xml
@@ -39,8 +39,8 @@
     <!-- Execution environment supported by Cloud Tools for Eclipse -->
     <requiredExecutionEnvironment>JavaSE-1.8</requiredExecutionEnvironment>
 
-    <tycho.version>1.2.0</tycho.version>
-    <tycho-extras.version>1.2.0</tycho-extras.version>
+    <tycho.version>1.1.0</tycho.version>
+    <tycho-extras.version>1.1.0</tycho-extras.version>
     <product.version.qualifier.suffix/>  <!-- 0-length string by default -->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <eclipse.target>photon</eclipse.target> <!-- the default build -->

